choreolib/src/main/java/choreo/auto/AutoFactory.java

```diff
@@ -203,7 +203,7 @@ public AutoRoutine newRoutine(String name) {
    * @return True if the alliance is known or alliance flipping is disabled.
    */
   private boolean allianceKnownOrIgnored() {
```

Comment:
This is only used in `newRoutine()`, so it probably doesn't need its own method.